athlinks-schema.ttl

```diff
@@ -174,4 +193,107 @@ athlinks:chipTime
 athlinks:agePercentage
   a owl:DataTypeProperty ;
   rdfs:domain athlinks:IntervalResult;
-  rdfs:range schema:Decimal .
+  rdfs:range xsd:decimal .
+
+athlinks:isHidden
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isVisible
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
```

Comment:
If we cannot define what it means to us in the new data model then we cannot add it, as it would be ambiguous. We would have to go through the process of interpreting what meaning those data points have, and then determine the proper way to store them in the new data model. Simply copying fields over puts us in a bad place for data unification. Which is one of the goals of this new data model.